### DISCUSSION TITLE
Installing oh-my-posh on macOS Ventura

### DISCUSSION THREAD

**Discussion by dpokluda** [JanDeDobbeleer/oh-my-posh#3016] [General]
I am following instructions from [macOS](https://ohmyposh.dev/docs/installation/macos):

```
brew install jandedobbeleer/oh-my-posh/oh-my-posh
```

but that gives me the following error:

```
Error: Your Command Line Tools (CLT) does not support macOS 13.
```

It says to update Command Line Tools (CLT), which I did:

```
softwareupdate --all --install --force
sudo rm -rf /Library/Developer/CommandLineTools
sudo xcode-select --install
```

But I am still seeing the same error. Is this a known issue?

**Comment by JanDeDobbeleer** [comment]
@dpokluda I'll check, but this is an issue with homebrew and not oh-my-posh if I'm not mistaken.

**Reply by JanDeDobbeleer** [reply]
i was able to upgrade using only `xcode-select --install`
